### ISSUE TITLE
NVDA Not Reading any Content of webpage

### ISSUE THREAD

**Issue by suhridsingh** [nvaccess/nvda#15199]
**NVDA is not reading any content of websites**
- Tried different websites on different browsers & incognito tabs as well
- Tried last two NVDA versions ie. 21.2 & 23.1, doesn't work
- NVDA Speech reader in Windows 10 having version 21.2 NOT reading content at all.
- Re-Installed - installed multiple times to do troubleshoot
- Nothing works

**Sys Conf.**
Wind 10/ Chrome Version 114.0.5735.248 (Official Build) (64-bit) / Edge Version 115.0.1901.183 (Official Build) (64-bit)

- also tried on friend's computer having Windows 11, NVDA Speech reader version 23.1 
-> Thre it is reading the content as "UNKNOWN",

**Comment by Brian1Gaff** [comment]
This sounds like Windows. I've seen this before, and assuming it did work, taking the machine back always fixed it, so never actually found a reason. Occasionally anti virus software can cause this if it removes something in a very bad way that changes the registry. Cannot comment about Windows 11 as I cannot run it.
 I'm assuming you ran the register fixing tool and rebooted the whole machine afterwards.
 
 I'm sure somebody will be along soon and ask you to fill in the template.
 Brian

-- 
***@***.*** 
Sent via blueyonder.(Virgin media) 
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.

**Comment by seanbudd** [comment]
Closing as duplicate of #15144.
In future please fill out the bug report template correctly
